Learn_go/网络通信/learn03-template/server/main.go

```
@@ -0,0 +1,54 @@
+	if err != nil {
+		fmt.Println(err)
+	}
+	fmt.Printf("准备开启服务器………")*/
+	handler1 := handler1{}
+	http.Handle("/handler1", &handler1)
+
+	s := http.Server{
+		Addr:    "127.0.0.1:8081",
+		Handler: nil,
+	}
+	s.ListenAndServe()
```

golangci-lint (1.64.8)

Error return value of `s.ListenAndServe` is not checked

(errcheck)